Apps/Contoso.WPF.Demo/App.xaml.cs

```diff
@@ -16,6 +19,22 @@ public partial class App
         protected override void OnStartup(StartupEventArgs e)
         {
             AppCenter.LogLevel = LogLevel.Verbose;
+            Crashes.GetErrorAttachments = (ErrorReport report) =>
+            {
+                byte[] fileContent = null;
+                var fileName = new FileInfo(Settings.Default.FileErrorAttachments).Name;
+                var mimeType = "";
+                if (File.Exists(Settings.Default.FileErrorAttachments))
+                {
+                    mimeType = MimeMapping.GetMimeMapping(Settings.Default.FileErrorAttachments);
+                    fileContent = File.ReadAllBytes(Settings.Default.FileErrorAttachments);
+                }
+                return new ErrorAttachmentLog[]
```

Comment:
Return attachments only when it exists. Adding an app log that says the file doesn't exist instead.

Reply:
Please also clean up the filename if it doesn't exist.